pkgs/development/compilers/zig/setup-hook.sh

```diff
@@ -20,8 +20,6 @@
 # https://github.com/NixOS/nixpkgs/pull/241741#issuecomment-1624227485
 # https://github.com/ziglang/zig/issues/14281#issuecomment-1624220653
 
-readonly zigDefaultFlagsArray=("-Drelease-safe=true" "-Dcpu=baseline")
```

Comment:
Instead of spreading the macro substitution around all the text, it is better to concentrate it here:

```bash
readonly zigDefaultFlagsArray=(@zig_default_flags@)
```